CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ Main (unreleased)
 
 - Flow: `prometheus.relabel` and `prometheus.remote_write` will now error if they have exited. (@ptodev)
 
+- Flow UI: Fix the layout for the object value in component detail page. (@xiyu95)
```

Comment:
Since the issue this is fixing was never part of a release, we wouldn't include it in bug fixes here